core/core.go

```diff
@@ -937,6 +980,17 @@ func (t *TextileNode) GetFile(path string, album *trepo.PhotoAlbum) ([]byte, err
 	return b, err
 }
 
+// GetFileBase64 returns data encoded as base64 under an ipfs path
```

Comment:
where does this get used?

Reply:
currently not used, almost deleted it but left around in case it makes sense to get meta this way